src/Console/CommandRunner.php

```diff
@@ -214,11 +214,6 @@ public function getEventManager(): EventManagerInterface
      */
     public function setEventManager(EventManagerInterface $eventManager)
     {
-        assert(
-            $this->app instanceof EventDispatcherInterface,
-            'Cannot set the event manager, the application does not support events.'
-        );
```

Comment:
I think you'll need this assertion to get stan tools to pass.

Reply:
I added an "actual" `instanceof` check instead